custom-payment-flow/server/node/server.js

```diff
@@ -30,7 +33,7 @@ app.get('/config', (req, res) => {
 });
 
 app.post('/create-payment-intent', async (req, res) => {
-  const { paymentMethodType, currency } = req.body;
+  const { currency, paymentMethodType, paymentMethodOptions } = req.body;
```

Comment:
I'm running into a similar issue when trying to add ACSS where I need to pass some payment method options. I'm not sure this is a safe way to handle tho. Wondering if we should construct the payment method options in this route switching on the payment method type...

```js
const params = {
  payment_method_types: [paymentMethodType],
  currency: currency,
  amount: 1999,
}
if (paymentMethodType === 'acss_debit') {
  params.payment_method_options = {
    acss_debit: {
      mandate_options: {
        payment_schedule: 'sporadic',
        transaction_type: 'personal',
      },
    },
  }
}
if (paymentMethodType === 'konbini'){
  params.payment_method_options = {
    konbini: {
      product_description: 'Tシャツ',
      expires_after_days: 3,
    },
  }
}
```

wdyt?

Reply:
Yes, you're right, thanks